tests/benchmarks/benchmark_cpu_small.py

```diff
@@ -355,17 +338,16 @@ def test_proximal_jac_atf(benchmark):
     prox.compile()
     x = prox.x(eq)
 
-    def run(x):
-        prox.jac_scaled(x, prox.constants).block_until_ready()
+    def run(x, prox):
+        prox.jac_scaled_error(x, prox.constants).block_until_ready()
 
-    benchmark.pedantic(run, args=(x,), rounds=15, iterations=1)
+    benchmark.pedantic(run, args=(x, prox), rounds=10, iterations=1)
 
 
 @pytest.mark.slow
 @pytest.mark.benchmark
 def test_proximal_freeb_compute(benchmark):
     """Benchmark computing free boundary objective with proximal constraint."""
-    jax.clear_caches()
```

Comment:
why remove this?

Reply:
This doesn't cause too much difference,(I can re add it) but technically only `run` is benchmarked, so clearing the cache here doesn't have much purpose.